src/index.ts

```diff
@@ -11,6 +11,7 @@
  * - if (typeof var === 'number') {}
  * - etc...
  */
+import "isomorphic-fetch";
```

Comment:
might want to add a comment explaining why this is being imported, when it's not being used